cloud/service-operations/vpc.md

```diff
@@ -33,9 +33,7 @@ VPC peering:
 *   Configure route table rules
 *   Configure security group and firewall rules
 
-By default, Timescale Cloud allows you to have three VPCs in each project. If
-you need more VPCs, [contact our support team][timescale-support] to ask for a
-quota increase.
+Timescale Cloud allows you to create as many VPCs in your project as you need to.
```

Comment:
We were not changing the number of VPCs per project. Rather we've allowed adding multiple peering connections for each VPC.

<img width="1449" alt="Screenshot 2023-01-18 at 14 57 35" src="https://user-images.githubusercontent.com/11886085/213190136-fed3a601-5e24-47dc-9b4d-294944858efd.png">

Reply:
The feature is currently behind the multi-vpc feature flag, in case you would like to try it out 🏁